.github/workflows/linux-workflow.yaml

```diff
@@ -115,7 +115,7 @@ jobs:
           mkdir -p ./ci-artifacts/out
           ./.github/scripts/releases/extract_build_linux.sh ./ci-artifacts/out ./
           pushd ci-artifacts
-          tar czf linux.tar.gz ./out
+          tar czf linux.tar.gz -C ./out .
```

Comment:
I forget what happened here... maybe this got changed and then reverted?
There was some issue with `tar` seeing its own output as it was going?

Reply:
It's a confusing history, I had something somewhat similar (still contained the out folder) https://github.com/open-goal/jak-project/pull/1326/files

And then it was changed slightly here https://github.com/open-goal/jak-project/commit/e8911758f0d07d70b4be2cbfc3ee052e4e263ad9

the `pushd/popd` is basically manually doing what `-C ...` is supposed to do.  I havn't had a problem on my machine but i can switch to that style if it's more consistent?